site/content/includes/installation/install-gateway-crds.md

```diff
@@ -0,0 +1,30 @@
+---
+docs:
+---
+
+{{<note>}}The [Gateway API resources](https://github.com/kubernetes-sigs/gateway-api) from the standard channel must be installed before deploying NGINX Gateway Fabric. If they are already installed in your cluster, please ensure they are the correct version as supported by the NGINX Gateway Fabric - [see the Technical Specifications](https://github.com/nginxinc/nginx-gateway-fabric/blob/main/README.md#technical-specifications).{{</note>}}
```

Comment:
Should the tech specs link be to docs.nginx.com?

Reply:
The docs.nginx.com link just sends it back to github right now.